packages/libs/src/sdk/services/Solana/programs/BaseSolanaProgramClient.ts

```diff
@@ -123,18 +123,21 @@ export class BaseSolanaProgramClient {
         )
       } else {
         const res = await this.connection.getRecentPrioritizationFees()
-        const orderedFees = res.map((r) => r.prioritizationFee).sort()
+        const orderedFees = res
+          .map((r) => r.prioritizationFee)
+          .sort((a, b) => a - b)
```

Comment:
Ah yes, one of the lost 20 commandments of JS: sort is pretty much useless on its own.... :-)